makejdk.sh

```diff
@@ -260,12 +260,9 @@ buildAndTestOpenJDKViaDocker()
 
 testOpenJDKInNativeEnvironmentIfExpected()
 {
-  if [[ ! -z $JTREG ]]; then
-    if [[ ! -z $JTREG_TEST_SUBSETS ]]; then
+  if [[ "${JTREG}" == "true" ]];
+  then
```

Comment:
@gdams this is the line that failed strangely, so best its clear and consistent (we already have scripts that now use "${...}" ), also see conversation https://github.com/AdoptOpenJDK/openjdk-build/pull/43#discussion_r110635712. Not too fused just lets follow one standard.